db/migrate/20230113094502_add_hal_id_to_university_persons.rb

```diff
@@ -0,0 +1,5 @@
+class AddHalIdToUniversityPersons < ActiveRecord::Migration[7.0]
+  def change
+    add_column :university_people, :hal_person_identifier, :string
```

Comment:
manque l'index si pas null

Reply:
j'ai pas compris. Ca peut être null, est-ce qu'il faut que j'ajoute un index à ton avis ? Tu penses à quoi ? L'optim ? On ne va pas chercher par ce champ à court terme

Reply:
je parle du fait d'indexer ça (comme c'est un identifiant utilisable en recherche l'indexer améliorera la recherche) mais de ne l'indexer que si la donnée est remplie pour ne pas que ça soit contreproductif.
En gros 
add_index : university_people, : hal_person_identifier, name: "index_university_people_on_ hal_person_identifier", where: "(hal_person_identifier IS NOT NULL)"

Reply:
pas de recherche sur les identifiants